.github/CONTRIBUTING.md

```diff
@@ -71,3 +71,6 @@ e.g.: `hotfix/1-<hotfix-name>` (whereas feature has been requested in _issue_ #1
 * __`release/<release-version>`__ - Branch where final adjustments/build will be made prior to delivery of a software product release. Branch name will be the version of the release to be delivered.
 
 * __`support/<theme-or-kind>`__ - Branch where support tasks related to the software will be performed, such as documentation development, configuration management corrections, and so on.
+
+* __`improvement/<improvement-name>`__ - Branch used for the improvement of functionalities or documentation project . If the improvement has been proposed by a _issue_ of the repository, the name should contain the _issue_ number.
```

Comment:
Tem alguma justificativa para este tipo de _branch_? Acredito que _feature_ e _bugfix_ encobrem esse _improvement_. E a ideia destes tipos era pra trabalhara com o git flow, vocês chegaram a olhar o git flow?

Reply:
We saw this release that feature and improvement are not the same thing and we feed that is necessary to create another type of branch tag

Reply:
So a good setup should be feature, bugfix and improvement.

Reply:
Do you agree about this @sconetto?

Reply:
Okay, fine for me, in the past was used these types to facilitate the workflow with git flow, but seems reasonable this new one!